src/components/Reports/Reports.test.js

```diff
@@ -46,13 +49,8 @@ describe('Reports page', () => {
       </DopplerIntlProvider>,
     );
 
-    await wait(() => getByText(fakeData[0].verified_date));
-
-    const verifiedDate = getByText(fakeData[0].verified_date);
-    const domain = getByText(fakeData[1].name);
-
-    expect(verifiedDate).toBeDefined();
```

Comment:
i didn't know no need expect in a test. I always saw it in the examples

Reply:
You said me that _getByText_ fails if the literal is not found, so, expected is redundant, right?

Reply:
yes but more clear with expect. for me at least 😄

Reply:
Mmmm, in general, I think that it is not a good idea having redundant code, in my opinion, it tends to be dangerous. 

But, if you want I can add these lines again, by the moment it is not so important (for me)

Reply:
@fcoronelmakingsense @cbernat let me know your opinion, in order to update it (or not) before merging.

Reply:
i'm already put my opinion but it's works the 2 ways

Reply:
Yes it is redundant, and yes it is clearer with the expect. Both ways work for me. I think is an issue of how we want to work.